app/services/diagnostics/move_inspector.rb

```diff
@@ -68,13 +96,31 @@ def generate
       ENDJOURNEYEVENTS
       if move.journeys.any?
         move.journeys.default_order.each do |journey| # NB use each to preserve sort order
-          @output << "#{journey.from_location.title} --> #{journey.to_location.title} (#{journey.id})\n"
+          @output << "#{journey.id}: #{journey.from_location} --> #{journey.to_location}\n"
```

Comment:
As you've overridden `.to_s` for locations this will now be quite a bit more wordy with nomis and guids shown for each location too - just wanted to check that was intentional?

Reply:
yep it was intentional, currently locations are rendered as:
```
> Location.first.to_s
=> "#<Location:0x00007fac213460a0>"
```

So this is an improvement and useful for general debugging:
```
> Location.first.to_s
=> "Huddersfield Youth Court (HUDRYC 00034e29-9ef8-4802-a2f5-09342bf9f174)"
```

The reference endpoints return location title and key so no changes there.